iothub/device/src/Authentication/AuthenticationWithTokenRefresh.cs

```diff
@@ -13,7 +13,10 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     public abstract class AuthenticationWithTokenRefresh : IAuthenticationMethod, IDisposable
     {
-        private readonly int _suggestedTimeToLiveSeconds;
+        private const int DefaultSasRenewalBufferPercentage = 15;
+        private static readonly TimeSpan s_defaultSasTimeToLive = TimeSpan.FromHours(1);
```

Comment:
Define the default ttl and buffer values in the base AuthenticationWithTokenRefresh class.